core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -363,11 +377,29 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
         emit SettleExpiredPosition(msg.sender, totalRedeemableCollateral.rawValue, tokensToRedeem.rawValue);
     }
 
+    /****************************************
+     *        GLOBAL STATE FUNCTIONS        *
+     ****************************************/
+
+    /**
+     * @notice After expiration of the contract the DVM is asked what the prevailing price at the
+     * time of expiration. Final fees are paid. Once price is resolved token holders can withdraw.
```

Comment:
nit: maybe reword this comment? After reading, it's a little difficult to understand which part of this the `expire` method does vs what happens after.

Reply:
ye, sorry about that. have fixed and pushed